practice/practice-02/src/sass/base/_typography.scss

```diff
@@ -0,0 +1,6 @@
+$sofia-pro-font-path: "../assets/fonts";
+
+@font-face {
+  src: url("#{$sofia-pro-font-path}/Sofia_Pro_Regular.otf") format("opentype");
```

Comment:
File name is not good, and pls read about font extention: https://design.tutsplus.com/articles/different-font-file-types-explained-ott-ttf-woff--cms-39047

Reply:
Fixed and I will read it, tks for sharing.